bower.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "marketplace-frontend": "7.0.1",
```

Comment:
Heh, `marketplace-frontend@7.0.1` depends on `commonplce@*`. So I guess that sidesteps the issue since it's so old.